### PR TITLE
display multisig type

### PR DESCRIPTION
**Pull request by lanchana**

### CODE REVIEW

File: src/components/list/WalletRow.tsx [hunk 1]
```diff
@@ -105,6 +107,9 @@ const WalletRow = ({wallet, onPress}: Props) => {
           </H5>
           {buildTestBadge(network, currencyName, isToken)}
         </Row>
+        <ListItemSubText>
+          {currencyAbbreviation.toUpperCase()} {multisig ? multisig : null}
+        </ListItemSubText>
         <ListItemSubText>{currencyAbbreviation.toUpperCase()}</ListItemSubText>
```
Comment (gabrielbazan7) [inline src/components/list/WalletRow.tsx:113]: Need a condition here otherwise both lines will be shown

Comment (gabrielbazan7) [reply src/components/list/WalletRow.tsx:113]:
```
        {multisig ? (
          <ListItemSubText>
            {currencyAbbreviation.toUpperCase()} {multisig}
          </ListItemSubText>
        ) : (
          <ListItemSubText>
            {currencyAbbreviation.toUpperCase()}
          </ListItemSubText>
        )}
```

Comment (gabrielbazan7) [reply src/components/list/WalletRow.tsx:113]:
```
        {multisig ? (
          <ListItemSubText>
            {currencyAbbreviation.toUpperCase()} {multisig}
          </ListItemSubText>
        ) : (
          <ListItemSubText>
            {currencyAbbreviation.toUpperCase()}
          </ListItemSubText>
        )}
```

**Comment by gabrielbazan7** [comment]
ACK

**Review by JohnathanWhite — APPROVED** [review]

**Pull request merged**